core/src/main/java/nl/nn/adapterframework/jdbc/migration/Migrator.java

```diff
@@ -50,18 +49,17 @@ public Migrator() {
 
 	@Override
 	public void configure() throws ConfigurationException {
-		throw new IllegalStateException("No configuration is specified!");
-	}
-
-	public void configure(Configuration configuration) throws ConfigurationException {
-		configure(configuration, null);
+		if(!(getApplicationContext() instanceof Configuration)) {
+			throw new IllegalStateException("context not instanceof configuration");
+		}
+		configure((Configuration) getApplicationContext(), null);
 	}
 
-	public synchronized void configure(Configuration configuration, String changeLogFile) throws ConfigurationException {
+	private synchronized void configure(Configuration configuration, String changeLogFile) throws ConfigurationException {
 		AppConstants appConstants = AppConstants.getInstance(configuration.getClassLoader());
 		setName("JdbcMigrator for configuration["+ configuration.getName() +"]");
-		if(StringUtils.isEmpty(getDatasourceName())) {	
-			setDatasourceName(appConstants.getString("jdbc.migrator.dataSource", null));
+		if(StringUtils.isEmpty(getDatasourceName())) {
+			setDatasourceName(appConstants.getString("jdbc.migrator.datasource", appConstants.getString("jdbc.migrator.dataSource", null)));
```

Comment:
Hier gebruik je als default de zojuist gevonden waarde, dat heeft geen zin.

Reply:
De oude property is `jdbc.migrator.dataSource` met grote S. Nu zoekt hij eerst naar datasource, en doet een fallback naar dataSource

Reply:
Ah! Misschien dat er even in een comment bij zetten, dat mensen die daar over heen lezen (zoals ik) het ook zien.
Zou je ook kunnen doen via constants

Reply:
Ik ben bezig met een vervolg pr, daar zit hij (nu) in!